Formula/igv.rb

```diff
@@ -1,30 +1,29 @@
 class Igv < Formula
   desc "Interactive Genomics Viewer"
   homepage "https://www.broadinstitute.org/software/igv"
-  url "https://data.broadinstitute.org/igv/projects/downloads/2.8/IGV_2.8.13.zip"
-  sha256 "95c90ac4327c5bc135cd0c94f66123930bd05050f3a64271b00cca5b52ae9fc6"
+  url "https://data.broadinstitute.org/igv/projects/downloads/2.9/IGV_2.9.4.zip"
+  sha256 "05717ec6d0386a5eafe93c8b1af371c0013355eae74312b78eae5fdd99fc5fc1"
+  license "MIT"
 
   livecheck do
     url "https://software.broadinstitute.org/software/igv/download"
     regex(/href=.*?IGV[._-]v?(\d+(?:\.\d+)+)\.zip/i)
   end
 
-  bottle :unneeded
-
   depends_on "openjdk"
 
   def install
     inreplace ["igv.sh", "igvtools"], /^prefix=.*/, "prefix=#{libexec}"
     bin.install "igv.sh" => "igv"
     bin.install "igvtools"
     libexec.install "igv.args", "lib"
-    bin.env_script_all_files libexec, JAVA_HOME: Formula["openjdk"].opt_prefix
+    bin.env_script_all_files libexec, Language::Java.overridable_java_home_env
   end
 
   test do
     assert_match "Usage:", shell_output("#{bin}/igvtools")
     assert_match "org/broad/igv/ui/IGV.class", shell_output("#{Formula["openjdk"].bin}/jar tf #{libexec}/lib/igv.jar")
     (testpath/"script").write "exit"
-    assert_match "Version", shell_output("#{bin}/igv -b script")
+    assert_match version.to_s, shell_output("#{bin}/igv --version")
```

Comment:
As mentioned, `igv` appears to open a GUI, which is causing issues with testcase.

I don't know if there is a better option than just running version check.

Based on trying to locally run `igv` , the GUI does seem to start up on both JDK15 and JDK11, and I was able to click on genome, download another genome, etc.